source/documentation/getting_started/get_started.erb

```diff
@@ -16,6 +16,24 @@ Once you have a personal account, you can access the GOV.UK PaaS admin tool for
 
 Contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you have any questions.
 
+### Trial and paid accounts
+
+When your department, agency or team first [signs up to the GOV.UK PaaS](https://www.cloud.service.gov.uk/signup), their account is a free trial account.
```

Comment:
"GOV.UK PaaS" does not need to be preceded with "the"

Reply:
To me, this reads like its a necessary step. A team can set up a paid account straight away.

We offer the trial account for evaluation if they need this

Reply:
agree and have changed